### ISSUE TITLE
Translate German

### ISSUE THREAD

**Issue by hume1991** [nextcloud/data_request#11]
Hello, 

can I translate in German?

Best regards 

Christian Huml

**Comment by blizzz** [comment]
Of course! The translations are done at transifex: https://www.transifex.com/nextcloud/nextcloud/data_request/

**Comment by hume1991** [comment]
Where should I integrate the file? (.po) must be converted?

Best regards

Christian Huml

**Comment by blizzz** [comment]
It happens all on transifex, see the link above.
